apps/emqx_rule_engine/src/emqx_rule_engine.erl

```diff
@@ -608,3 +615,16 @@ find_type(ResId) ->
 
 alarm_name_of_resource_down(Type, ResId) ->
     list_to_binary(io_lib:format("resource/~s/~s/down", [Type, ResId])).
+
+start_reinitial_loop(Id) ->
+    _Pid = spawn(fun() -> resource_check(Id) end).
+
+resource_check(Id) ->
+    timer:sleep(60000),
+    case emqx_rule_registry:find_resource_params(Id) of
```

Comment:
This is not necessary as the resource must have not initialized if we goes into this function.